flink-ml-lib/src/main/java/org/apache/flink/ml/feature/interaction/Interaction.java

```diff
@@ -0,0 +1,178 @@
+/*
```

Comment:
Let's add documents to the description of this PR, explaining that nominal features should be one-hot encoded before inputting to this algorithm. You can refer to #139 's description.